app/javascript/store/index.js

```diff
@@ -157,7 +157,14 @@ const store = new Vuex.Store({
   getters: {
     productsAsArray: state => (Object.keys(state.products).map(key => ({ id: key, ...state.products[key] }))),
     onSaleProducts: (state, getters) => (getters.productsAsArray
-      .filter(product => product.for_sale)),
+      .filter(product => product.for_sale))
+      .sort((a, b) => a.name.localeCompare(b.name)),
+    groupBySnack: (state, getters) => (getters.onSaleProducts
+      .filter(product => product.category == "snacks")),
+    groupByDrink: (state, getters) => (getters.onSaleProducts
+      .filter(product => product.category == "drinks")),
```

Comment:
Expected '===' and instead saw '=='  eqeqeq<br>Strings must use singlequote         quotes